### ISSUE TITLE
Resource conflict with app_name string

### ISSUE THREAD

**Issue by vovkab** [requery/sqlite-android#7]
Most applications use `app_name` resource as application name, right now it is conflicting with our resources too.

Usually it is a good idea to use prefixes for library resources to make sure they don't conflict with main application, i.e. use `__requery_app_name` instead of `app_name`.

If someone have the same issue, until it is fixed you can use manifest merger option to replace app_label:

```
<application
  ...
  tools:replace="label"
  >
```

More details is here: http://developer.android.com/tools/building/manifest-merge.html

**Comment by trevjonez** [comment]
This should have been fixed by [this commit](https://github.com/requery/sqlite-android/commit/deadcc5cb866d60721c141bc997fbdcc24fbbf3a)

Are you using the latest release? (3.12.1)

**Comment by vovkab** [comment]
Nice thanks!

I'm using the one from main page:

```
dependencies {
    compile 'io.requery:sqlite-android:3.12.0'
}
```

I'll try 3.12.1

**Comment by vovkab** [comment]
Bumped version here #8

**Comment by npurushe** [comment]
Yup this was fixed in 3.12.1
